daemon/proto/40swapd.proto

```diff
@@ -2,15 +2,15 @@ syntax = "proto3";
 
 option go_package = "./rpc";
 
-service 40Swap {
+service SwapService {
   // RPC methods
-  rpc SwapOut(SwapOutRequest) returns (SwaptOutResponse);
+  rpc SwapOut(SwapOutRequest) returns (SwapOutResponse);
 }
 
 // Message definitions
 message SwapOutRequest {
   string chain = 1;
```

Comment:
Let's make this an enum:
- BITCOIN, LIQUID
- BTC, LBTC

Which one do we like more? 🤔

Reply:
I guess no 1, as the other are tickers